samples/metaapp/src/main/sqldelight/com.sourcepointmeta.metaapp.db/11.sqm

```diff
@@ -24,7 +24,7 @@
 -- COMMIT;
```

Comment:
github is showing we moved the file  `.../10.sqm` to `.../11.sqm`. Did you mean to copy the file or to move? Or is it just GitHub showing something wrong?

Reply:
this is the migration of SQLDelight library used when you add a new field in a table

Reply:
Got it. But did it create a new migration file or it moved the previous one to the new?
GitHub shows it renamed the file `10.sqm` to `11.sqm`.

Reply:
In this case creating a new one or renaming the previous one, doesn’t matter